src/main/java/seedu/eduke8/ui/Ui.java

```diff
@@ -492,6 +495,12 @@ public void printListOfBookmarkedQuestions(BookmarkList bookmarks) {
         printMessage(MESSAGE_BOOKMARK_LIST + System.lineSeparator() + list);
     }
 
+    public void printDeletedBookmarkIndicator(int deleteIndex) {
```

Comment:
Shouldn't variables be named using nouns and note with a verb? A better way of naming `deleteIndex` could be `indexToDelete`.